assets/src/edit-story/components/canvas/displayElement.js

```diff
@@ -140,6 +161,7 @@ DisplayElement.propTypes = {
   previewMode: PropTypes.bool,
   element: StoryPropTypes.element.isRequired,
   page: StoryPropTypes.page,
+  animationMode: PropTypes.bool,
```

Comment:
Nit, since we're not using modes, should this be named `enablePreviewAnimations` or something like that?  So it's clear (by name) that it's a boolean?

Reply:
yea I like that better. was just following the naming pattern that was already used for `previewMode`